swig/include/ogr.i

```diff
@@ -871,10 +871,15 @@ public:
   char const *name;
 %mutable;
 
+
   ~OGRDataSourceShadow() {
     OGRReleaseDataSource(self);
   }
 
+  OGRErr Close() {
```

Comment:
should return a CPLErr or some CPLErr -> OGRErr translation should be done

Reply:
changed to `CPLErr`